.gitignore

```diff
@@ -146,6 +146,7 @@ tagCountHistogram2.png
 uniqueHistogram.png
 test-files/cmd/current
 test-files/DcTigerRoadsHighwayExactMatchInputs.osm
+test-files/DcTigerRoadsFrechetInput.osm
```

Comment:
How is test-files/DcTigerRoadsFrechetInput.osm different from test-files/DcTigerRoads.osm?